### PR TITLE
WIP: fix/warnings-checkstyle

### PR DESCRIPTION
**Pull request by PabloGalegoC**
Fix Google Java code style warnings

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -56,40 +56,68 @@
...
+                    <execution>
+                        <id>quality-metric-spotbugs</id>
+                        <phase>verify</phase>
+                        <goals>
+                            <goal>check</goal>
+                        </goals>
+                    </execution>
+                </executions>
+            </plugin>
             <plugin>
```
Comment (franrua) [inline pom.xml]: If this branch was created for fix warnings on checkstyle, why you add spotbugs? I think this must go on another PR

File: backend/src/main/java/com/shekhargulati/app/api/PingResource.java [hunk 2]
```diff
@@ -11,10 +11,15 @@
...
+   * Ping method.
+   *
+   * @return Response entity.
+   */
+  @GetMapping(path = "/ping")
+  public ResponseEntity<Map<String, String>> ping() {
+    Map<String, String> map = new HashMap<>();
+    map.put("ping", "pong");
+    return new ResponseEntity<>(map, HttpStatus.OK);
+  }
```
Comment (franrua) [inline backend/src/main/java/com/shekhargulati/app/api/PingResource.java]: I assume you removed some comments. If you do it must search in commit history and get it back

**Comment by franrua** [comment]
> Previous commits from branch 'enhancement/update-plugins' are needed since it is in that branch where the google checks were added.

The application of the different modifications to pom files has an order. You can solve issues and then update plugins after add the sporbugs or checkstyle plugins. The only thing that you don't must do is apply several actions in unique PR, beacause if anything go wrong you rollback a lot of changes.

**Comment by PabloGalegoC** [comment]
In the comment I left above I explain the reason: The changes made in this branch are only the ones in the last commit. The thing is that there were changes made in branch 'enhancement/update-plugins' (namely, update checkstyle) that were needed in this branch and weren't yet merged in master, so this branch was created from 'enhancement/update-plugins' instead of master.
